### ISSUE TITLE
ShortClassName should has minimum 3 symbols

### ISSUE THREAD

**Issue by dgroup** [dgroup/arch4u-pmd#136]
### Summary
| Key            |  Value |
|:---------------|:------|
| Rule           | ShortClassName |
| arch4u version       | 0.1.1 |

### Your rule definition
```xml
default rule configuration from pmd-java:6.44.0
```

### Describe the defect or false/positive rule violation
...
```java
class Arg { // violation

}
```
